saleor/graphql/views.py

```diff
@@ -294,7 +294,7 @@ def format_error(cls, error):
         else:
             unhandled_errors_logger.error("A query failed unexpectedly", exc_info=exc)
 
-        result["extensions"] = {"exception": {"code": type(exc).__name__}}
+        result["plugins"] = {"exception": {"code": type(exc).__name__}}
```

Comment:
These are GraphQL's error extensions, you can read about them here: http://spec.graphql.org/June2018/#sec-Errors. They allow us to pass additional data in the error responses. So we must not change it here and below.